src/Polly.Core/CircuitBreaker/BrokenCircuitException.cs

```diff
@@ -41,6 +57,15 @@ public BrokenCircuitException(string message, Exception inner)
     {
     }
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="BrokenCircuitException"/> class.
+    /// </summary>
+    /// <param name="message">The message that describes the error.</param>
+    /// <param name="inner">The inner exception.</param>
+    /// <param name="retryAfter">The time during which the circuit will be broken.</param>
+    public BrokenCircuitException(string message, Exception inner, TimeSpan retryAfter)
```

Comment:
```suggestion
    public BrokenCircuitException(string message, TimeSpan retryAfter, Exception inner)
```

keep the inner exception as last parameter